frontend/src/component/layout/MainLayout/NavigationSidebar/NavigationSidebar.tsx

```diff
@@ -0,0 +1,218 @@
+import { Link } from 'react-router-dom';
+import SearchIcon from '@mui/icons-material/Search';
+import PlaygroundIcon from '@mui/icons-material/AutoFixNormal';
+import InsightsIcon from '@mui/icons-material/Insights';
+import Accordion from '@mui/material/Accordion';
+import AccordionSummary from '@mui/material/AccordionSummary';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import AccordionDetails from '@mui/material/AccordionDetails';
+import IntegrationsIcon from '@mui/icons-material/IntegrationInstructionsOutlined';
+import EnvironmentsIcon from '@mui/icons-material/CloudOutlined';
+import ContextFieldsIcon from '@mui/icons-material/AccountTreeOutlined';
+import SegmentsIcon from '@mui/icons-material/DonutLargeOutlined';
+import TagTypesIcon from '@mui/icons-material/LabelImportantOutlined';
+import ApplicationsIcon from '@mui/icons-material/AppsOutlined';
+import CustomStrategiesIcon from '@mui/icons-material/ExtensionOutlined';
+import UsersIcon from '@mui/icons-material/GroupOutlined';
+import ServiceAccountIcon from '@mui/icons-material/SmartToyOutlined';
+import GroupsIcon from '@mui/icons-material/GroupsOutlined';
+import RoleIcon from '@mui/icons-material/AdminPanelSettingsOutlined';
+import ApiAccessIcon from '@mui/icons-material/KeyOutlined';
+import SingleSignOnIcon from '@mui/icons-material/AssignmentOutlined';
+import NetworkIcon from '@mui/icons-material/HubOutlined';
+import MaintenanceIcon from '@mui/icons-material/BuildOutlined';
+import BannersIcon from '@mui/icons-material/PhotoOutlined';
+import InstanceStatsIcon from '@mui/icons-material/QueryStatsOutlined';
+import LicenseIcon from '@mui/icons-material/ReceiptLongOutlined';
+import InstancePrivacyIcon from '@mui/icons-material/ShieldOutlined';
+import LoginHistoryIcon from '@mui/icons-material/HistoryOutlined';
+import EventLogIcon from '@mui/icons-material/EventNoteOutlined';
+import { ReactComponent as ProjectIcon } from 'assets/icons/projectIconSmall.svg';
+import type { FC } from 'react';
+
+export const StyledProjectIcon = styled(ProjectIcon)(({ theme }) => ({
+    fill: theme.palette.neutral.main,
+    stroke: theme.palette.neutral.main,
+    // same as built-in icons
+    width: theme.spacing(3),
+    height: theme.spacing(3),
+    fontSize: theme.spacing(3),
+}));
+
+const StyledListItem: FC<{ href: string; text: string }> = ({
+    href,
+    text,
+    children,
+}) => {
+    return (
+        <ListItem disablePadding>
+            <ListItemButton dense={true} component={Link} to={href}>
+                <ListItemIcon sx={(theme) => ({ minWidth: theme.spacing(4) })}>
+                    {children}
+                </ListItemIcon>
+                <ListItemText primary={text} />
+            </ListItemButton>
+        </ListItem>
+    );
+};
+
+export const StyledBox = styled(Box)(({ theme }) => ({
+    backgroundColor: theme.palette.background.paper,
+    pt: theme.spacing(3),
+    pb: theme.spacing(3),
+    minHeight: '95vh',
```

Comment:
Is this accurate?

Reply:
I'm trying to avoid a situation with a jumping footer